spec/order_spec.rb

```diff
@@ -2,12 +2,16 @@
 
 describe Order do
   let(:dish) { double("dish") }
-  let(:menu) { double "menu" }
-  subject { described_class.new(menu) }
+  let(:menu) { double("menu") }
+  let(:order) { double("order") }
```

Comment:
Great use of doubles! Something which still confuses me so much but this looks great!